grpc/examples/README.md

```diff
@@ -0,0 +1,7 @@
+## Language known issues
+
+### Go
+
+- Always requires the `content-type` of the payload to be set to `application/grpc+flatbuffers`
+
+example: `.SayHello(ctx, b, grpc.CallContentSubtype("flatbuffers"))`
```

Comment:
I added in the root examples as a reference to all the known issues regarding the compatibility of flatbuffers and grpc.